src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -171,10 +171,10 @@ export default (domain, domains, global) => {
         serverConfig.push(['# logging', '']);
 
         if (domain.logging.accessLog.computed)
-            serverConfig.push(['access_log', getAccessLogDomainPath(domain, global)]);
-
+            serverConfig.push(['access_log', getAccessLogDomainPath(domain, global) + `${global.logging.cloudflare.computed ? ' cloudflare' : ''}`]);
```

Comment:
```suggestion
            serverConfig.push(['access_log', getAccessLogDomainPath(domain, global) + (global.logging.cloudflare.computed ? ' cloudflare' : '')]);
```
No need for the template literal, just use the raw ternary